docs/source/models/intervention_models/neonatal/cpap_intervention.rst

```diff
@@ -126,13 +136,20 @@ Where,
     - Relative risk of CPAP intervention on RDS mortality 
     - 0.53 (95% CI 0.34-0.83). Uncertaintly interval implemented as parameter uncertainty following a lognormal distribution
     - `2020 Cochrane review <https://pmc.ncbi.nlm.nih.gov/articles/PMC8094155/>`_. Note that this effect was measured for all cause mortality.
+  * - :math:`\text{RR}_\text{ACS}`
+    - Relative risk of ACS intervention on RDS mortality 
+    - Refer to :ref:`ACS intervention page <acs_intervention>` for this effect size.
+    - Only to be included in PAF calculation if simulant is within the gestational age range that is eligible for ACS (26-33 weeks).
```

Comment:
To me, this implies that it is _not_ true that "we assume the same simulants have access to (i.e., are exposed to) CPAP and ACS", since it sounds like there are simulants outside the gestational age range who could receive CPAP but not ACS. Am I missing something?